Branch-SDK/src/main/java/io/branch/referral/Branch.java

```diff
@@ -625,6 +625,10 @@ private static Branch getBranchInstance(@NonNull Context context, boolean isLive
                 branchReferral_.linkCache_.clear();
                 branchReferral_.requestQueue_.clear();
             }
+
+            if (BranchUtil.getEnableFacebookLinkCheck(context)) {
+                branchReferral_.enableFacebookAppLinkCheck();
```

Comment:
Want to double check if this is equivalent to [ReactNativeImplementation](https://github.com/BranchMetrics/react-native-branch-deep-linking-attribution/blob/staging/android/src/main/java/io/branch/rnbranch/RNBranchModule.java#L740) in terms of lifecycle timing.